libsentrykube/service.py

```diff
@@ -112,9 +124,109 @@ def get_service_value_overrides(
 
         with open(service_override_file, "rb") as f:
             values = yaml.safe_load(f)
+
+        # Try to load the config only after {cluster_name}.yaml was successfully loaded because the common override
+        # only makes sense if there is a cluster config
+        common_override_values = get_common_regional_override(
+            service_name, region_name, external
+        )
+
+        if common_override_values:
+            deep_merge_dict(common_override_values, values)
+            return common_override_values
+
+        return values
     except FileNotFoundError:
-        values = {}
-    return values
+        return {}
+
```

Comment:
Each of these methods (`get_service_values`, `get_service_value_overrides`, etc.) loads a level of overrides only. Then these are layered in the right order in `_consolidate_variables`. This is intentional so that we separate the logic that loads one level of override from the one that defines the order and make the order apparent.

Please do not merge different levels of overrides in the same method and create a new one instead. The merging should only happen in `_consolidate_variables` only. If you find a lot of duplication feel free to factor that logic out.

Reply:
Thanks for the review, those are valid points. I will split it up and move the logic